src/Parser/ParserOptions.cs

```diff
@@ -68,6 +71,20 @@ public void SetupXcode()
 
         public void SetupIncludes()
         {
+            // Search for the available Clang versions
+            var location = System.Reflection.Assembly.GetExecutingAssembly().Location;
```

Comment:
Import System.Reflection instead, we only use qualified names in the generated code or to avoid conflicts.